### PR TITLE
Title changed - removed macOS and put Host Computer instead

### PR DESCRIPTION
**Pull request by cpholguera**
Changed macOS device to Host computer in the title so we can also put it in the Android chapter as "#### Recommended Tools - Host Computer". Also because it is not 100% required but recommended. We don't need a mac to run Frida, objection (to the device, not patching), Ghidra, burp, etc.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 4a739f8523c121971fb465e9bf3a6ba7f951ec71: Broken link result: [!] Error(s) found by the Links validator: 5 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06b-Basic-Security-Testing.md [hunk 1]
```diff
@@ -162,9 +162,9 @@ $ apt-get install adv-cmds
 
 On an iOS device you cannot make data connections anymore after 1 hour of being in a locked state, unless you unlock it again due to the USB Restricted Mode, which was introduced with iOS 11.4.1
 
-#### Recommended Tools - macOS Device
+#### Recommended Tools - Host Computer
 
-In order to analyze iOS apps, you should use a macOS device and install the following tools we'll be referring throughout the guide:
+In order to analyze iOS apps, you should install the following tools on your host computer. We'll be referring them throughout the guide. Please note that a great number of them will require a macOS system in order to run and therefore using a macOS computer is normally the recommendation when dealing with iOS apps.
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: referring them -> referring to them

Comment (sushi2k) [reply Document/0x06b-Basic-Security-Testing.md]: Let's get rid of system and just use macOS:

them will require a macOS system -> them will require macOS

Comment (cpholguera) [reply Document/0x06b-Basic-Security-Testing.md]: done, thanks Sven

**Comment by OMTGreleaser** [comment]
Results for commit f4ddbc19f90a371fa797a55042598ede3fc3a489: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06b-Basic-Security-Testing.md [hunk 2]
```diff
@@ -162,9 +162,9 @@ $ apt-get install adv-cmds
 
 On an iOS device you cannot make data connections anymore after 1 hour of being in a locked state, unless you unlock it again due to the USB Restricted Mode, which was introduced with iOS 11.4.1
 
-#### Recommended Tools - macOS Device
+#### Recommended Tools - Host Computer
 
-In order to analyze iOS apps, you should use a macOS device and install the following tools we'll be referring throughout the guide:
+In order to analyze iOS apps, you should install the following tools on your host computer. We'll be referring to them throughout the guide. Please note that a great number of them will require a macOS in order to run and therefore using a macOS computer is normally the recommendation when dealing with iOS apps.
```
Comment (sushi2k) [inline Document/0x06b-Basic-Security-Testing.md]: a macOS -> macOS

Comment (cpholguera) [reply Document/0x06b-Basic-Security-Testing.md]: done

**Comment by OMTGreleaser** [comment]
Results for commit 8cd754ce14eb30b4632b63087844082a71b97d30: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**